botorch/acquisition/fixed_feature.py

```diff
@@ -20,7 +22,31 @@
 from torch.nn import Module
 
 
-class FixedFeatureAcquisitionFunction(AcquisitionFunction):
+class AcquistionFunctionWrapperMixin(ABC):
```

Comment:
This is a very generic name for something rather specific, can name this differently? Like `PendingPointAcqfWrapperMixin` (or some better name)?

Reply:
Ah I'll remove this. The set_X_pending functionality was not as generic as I thought